bcp/client/src/webpack/helper/toGeoJSON.js

```diff
@@ -0,0 +1,209 @@
+                    category: arenaAsset.category,
+                    make: arenaAsset.make,
+                    model: arenaAsset.model,
+                    airframe: arenaAsset.airframe
+                }
+            };
+            pushFeatureData(aircraftGeoJSON, newFeature);
+        } else {
+            const newFeature = {
+                type: "Feature",
+                geometry: {
+                    type: "Point",
+                    coordinates: [
+                        location.longitude,
+                        location.latitude
+                    ]
+                },
+                properties: {
+                    ...location,
+                    imeiMatch: false,
+                }
+            };
+            pushFeatureData(aircraftGeoJSON, newFeature);
+        }
+    });
+    cb(aircraftGeoJSON);
+};
+export const fetchEquipmentGeoJSON = async (cb) => {
+    const {equipment, equipmentLocations} = await toGeoJSON();
+    const equipmentGeoJSON = {
+        type: 'FeatureCollection',
+        features: []
+    };
+    equipmentLocations.forEach(location => {
+        const arenaAsset = equipment.find(value => value.imei === location.imei);
+
+        if (arenaAsset) {
+            const newFeature = {
+                type: "Feature",
+                geometry: {
+                    type: "Point",
+                    coordinates: [
+                        location.longitude,
+                        location.latitude
+                    ]
+                },
+                properties: {
+                    ...location,
+                    imeiMatch: arenaAsset.imei,
+                    searchText: createSearchText(arenaAsset),
+                    callsign: arenaAsset.callsign,
+                    registration: arenaAsset.registration,
+                    dispatch_contact: arenaAsset.operator.name,
+                    dispatch_phone: arenaAsset.operator.operationalContact,
+                    dispatch_email: arenaAsset.operator.email,
+                    description: arenaAsset.description,
+                    fuelType: arenaAsset.fuelType
+                }
+            };
+            pushFeatureData(equipmentGeoJSON, newFeature);
+        } else {
+            const newFeature = {
+                type: "Feature",
+                geometry: {
+                    type: "Point",
+                    coordinates: [
+                        location.longitude,
+                        location.latitude
+                    ]
+                },
+                properties: {
+                    ...location,
+                    imeiMatch: false,
+                }
+            };
+            pushFeatureData(equipmentGeoJSON, newFeature);
+        }
+    });
+    cb(equipmentGeoJSON);
+};
+
+export const fetchIncidentGeoJSON = (cb) => {
+    fetch('/incidents.json').then(res => res.json()).then(
+        incidentJSON => {
+            cb(incidentJSON);
+        }
+    );
+};
```

Comment:
It looks like all the callbacks in this module can be replaced with a returned promise.